common/main/java/com/couchbase/lite/MutableArray.java

```diff
@@ -38,41 +38,45 @@
     /**
      * Constructs a new empty Array object.
      */
-    public MutableArray() {
+    public MutableArray() { super(); }
+
+    /**
+     * Creates a new MutableArray with content from the passed List.
+     * Allowed value types are List, Date, Map, Number, null, String, Array, Blob, and Dictionary.
+     * The List and Map must contain only the above types.
+     *
+     * @param data the document content list
+     */
+    public MutableArray(@NonNull List<Object> data) {
         super();
+        setData(data);
     }
 
     /**
-     * Constructs a new Array object with an array content. Allowed value types are List, Date,
-     * Map, Number, null, String, Array, Blob, and Dictionary. The List and Map must contain
-     * only the above types.
+     * Creates a new MutableArray with content from the passed JSON string.
      *
-     * @param data the array object.
+     * @param json the array content as a JSON string.
      */
-    public MutableArray(List<Object> data) {
```

Comment:
Should we remove this constructor??

Reply:
Yeah... I don't think the spec suggests removing this.  It seems consistent with the constructors for Document and Dictionary objects.